prism/src/main/java/org/apache/falcon/service/FeedSLAMonitoringService.java

```diff
@@ -269,10 +260,9 @@ public void destroy() throws FalconException {
     public void makeFeedInstanceAvailable(String feedName, String clusterName, Date nominalTime) {
         LOG.info("Removing {} feed's instance {} in cluster {} from pendingSLA", feedName,
                 clusterName, nominalTime);
-        Pair<String, String> feedCluster = new Pair<>(feedName, clusterName);
         // Slas for feeds not having sla tag are not stored.
-        if (pendingInstances.get(feedCluster) != null) {
-            pendingInstances.get(feedCluster).remove(nominalTime);
+        if (CollectionUtils.isEmpty(MONITORING_JDBC_STATE_STORE.getNominalInstances(feedName, clusterName))){
```

Comment:
for readability and efficiency it is better to store the result in a local variable and use that in test for if condition and below.